Cargo.toml

```diff
@@ -42,8 +42,8 @@ hyper-util = "0.1"
 
 ic-cdk = "0.13"
 ic-cdk-macros = "0.13"
-ic-agent = "0.37"
-ic-utils = "0.37"
+ic-agent = "0.38.2"
```

Comment:
We should use `0.38` for both of these crates, that will lower the possibility of clashes between crate versions for consumers of libraries in this repo.

Reply:
I believe there is a 0.38.1 version that doesn't include the change we need. But I'll double check.

Reply:
This is 0.38.0 release: https://github.com/dfinity/agent-rs/blob/729398880545a1f49a4c6286d0dd0c94254b236f/ic-agent/src/agent/mod.rs#L1187

Maybe we can bump it at the next minor version bump.